Subject: don't break build while cross compiling
From: Robert Schwebel <r.schwebel@pengutronix.de>

When cross compiling it may not really matter if we have the docbook xsl
stuff (only needed for documentation). So disable the check in that
case.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: elektra-0.7.0-r1618/configure.ac
===================================================================
--- elektra-0.7.0-r1618.orig/configure.ac
+++ elektra-0.7.0-r1618/configure.ac
@@ -301,7 +301,7 @@ PKG_PROG_PKG_CONFIG
 # xsltproc
 AC_PATH_PROG(xsltproc, xsltproc, "no")
 # Check if have style-sheet too
-if test "x$default_docbook" != "xno"; then
+if test "x$default_docbook" != "xno" && test "x$cross_compiling" = "xno"; then
 	AC_CHECK_FILE([$default_docbook/manpages/docbook.xsl],
 		      [dbroot=$default_docbook],
 		      [dbroot="no"])
